.github/workflows/validation-coding-standards.yml

```diff
@@ -0,0 +1,47 @@
+name: 'Validation / Coding Standards'
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+    name: PHP
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Composer Install
+        run: composer install --prefer-dist --optimize-autoloader
+
+      - name: PHPCS
+        run: composer lint
+
+  css:
+    runs-on: ubuntu-latest
+    name: CSS
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: php-actions/composer@v5
```

Comment:
@EvanHerman what is the reason for using this external Action? 

In practice, we will use little (if any) external Actions because they cannot be used in our private Corp GitHub. We must follow that same company restriction for our Open Source GoDaddy GitHub.

Reply:
Only used the external actions because it was cleaner. But I actually don't think we need this at all. I re-tested and pushed up some changes removing this step altogether.